termino/termino.py

```diff
@@ -55,14 +63,27 @@ async def red_delete_data_for_user(self, **kwargs):
         """Nothing to delete"""
         return
 
+    async def startup(self):
+        await self.bot.wait_until_red_ready()
+        conf = await self.config.all()
+        maybe_channel = conf.get("restart_channel", None)
+        await self.config.restart_channel.clear()
+        # Using the walrus operator we can check if the channel is originally None
```

Comment:
I am unfamiliar completely with the walrus operator and it is something I need to look into, but I trust that it works here, so no problems there.